qbraid_qir/qiskit/visitor.py

```
@@ -0,0 +1,331 @@
+class BasicQiskitVisitor(QuantumCircuitElementVisitor):
+    """A visitor for basic Qiskit circuit elements.
+
+    This class traverses and converts Qiskit circuit elements to QIR.
+
+    Args:
+        initialize_runtime: If True, quantum runtime will be initialized. Defaults to True.
+        record_output: If True, output of the circuit will be recorded. Defaults to True.
+        emit_barrier_calls: If True, barrier instructions will be emitted. Defaults to False.
+    """
+
+    def __init__(
```

Ruff (0.15.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)